gitreceive/receiver/flynn-receive.go

```diff
@@ -189,23 +199,25 @@ Options:
 			Processes: map[string]int{"web": 1},
 		}
 
-		watcher, err := client.WatchJobEvents(app.ID, release.ID)
-		if err != nil {
-			log.Fatalln("Error streaming job events", err)
-			return
-		}
-		defer watcher.Close()
```

Comment:
We should continue to only watch for job events here on initial push (in which case the deployment didn't do anything and we need to scale up), and then on subsequent pushes we should print deployment events (which are waited on [here](https://github.com/flynn/flynn/blob/master/controller/client/client.go#L495-L540)).